README.md

```diff
@@ -51,6 +51,8 @@ The cmake configuration options change which execution patterns and memory space
   - __ENABLE_OPENMP__ Allow use of openmp and enable test combinations using openmp
   - __ENABLE_CUDA__  Allow use of cuda and enable test combinations using cuda
   - __ENABLE_RAJA__  Allow use of RAJA performance portability library
+  - __ENABLE_CALIPER__ Allow use of the Caliper performance profiling library
+  - __ENABLE_ADIAK__ Allow use of the Adiak library for recording program metadata
```

Comment:
Are there any version requirements to document"

Reply:
Should work with any Adiak version and any recent Caliper version (v2.2+, which was released in 2019).